partner_event/views/event_registration_view.xml

```diff
@@ -1,31 +1,25 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!-- Copyright 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
```

Comment:
Leave this untouched please. Review https://github.com/OCA/maintainer-tools/pull/197#issue-158635253.